src/bootstrap/compile.rs

```diff
@@ -198,6 +198,10 @@ pub fn rustc<'a>(build: &'a Build, target: &str, compiler: &Compiler<'a>) {
     if !build.unstable_features {
         cargo.env("CFG_DISABLE_UNSTABLE_FEATURES", "1");
     }
+    // Flag that rust llvm is in use
+    if build.config.target_config.get(target).is_none() {
```

Comment:
The logic for parsing `./configure`'s `--llvm-root` option [lives here](https://github.com/rust-lang/rust/blob/c2b03f8ba9c3f2622ea9fa6e2453c57aa1f8e6ab/src/bootstrap/config.rs#L352-L357) so it only sets it for one target but it should be there?